optimum/bettertransformer/models/base.py

```diff
@@ -115,3 +143,47 @@ def forward_checker(self, *args, **kwargs):
                 "Training is not supported for `BetterTransformer` integration.",
                 " Please use `model.eval()` before running the model.",
             )
+
+    def _recurse_setattr(self, module, name, value):
+        r"""
+        A wrapper function to recursively set attributes to a module.
+        """
+        if "." not in name:
+            setattr(module, name, value)
+        else:
+            name, rest = name.split(".", 1)
+            self._recurse_setattr(getattr(module, name), rest, value)
```

Comment:
can you move this to utils?

Reply:
fixed in [aefc535](https://github.com/huggingface/optimum/pull/676/commits/aefc535d66dc233ca70c71ff40ebb178ad914c0f)